extensions/amp-analytics/0.1/events.js

```diff
@@ -470,6 +474,19 @@ export class VideoEventTracker extends EventTracker {
         return;
       }
 
+      if (normalizedType === VideoAnalyticsEvents.SECONDS_PLAYED) {
+        if (interval) {
+          intervalCounter++;
+          if (intervalCounter % interval !== 0) {
+            return;
+          }
+        } else {
+          user().error(TAG, 'video-seconds-played requires interval spec ' +
```

Comment:
possible to move the validity check up maybe at the time `videoSpec['interval']` is read? I assume `eventType` can be used to detect whether interval is required or not.